app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -84,15 +84,15 @@ def activity_session_params
                   :completed_at,
                   :activity_uid,
                   :anonymous,
-                  concept_tag_results_attributes:  concept_tag_keys)
+                  concept_results_attributes:  concept_tag_keys)
       .merge(data: @data).reject {|k,v| v.nil? }
   end
 
   # Grab a list of all the arbitrarily-named keys that are provided in the concept tag results payload.
   # Returns a list of symbols, e.g. [:student_input, :wpm]
-  def concept_tag_result_allowed_keys
-    if params[:concept_tag_results_attributes]
-      params[:concept_tag_results_attributes].reduce [] do |acc, hash|
+  def concept_result_allowed_keys
```

Comment:
Yeah strong parameters are kind of crazy in Rails. That hash is freeform (no specific required keys) and the code sample I found for whitelisting the contents of specific keys didn't seem to work, so the next best recommended solution seemed to involve whitelisting every key in the hash. Not a fan though.